package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "vui-table",
+  "version": "1.0.0",
+  "description": "Mixins and CSS for applying table styles",
+  "scripts": {
+    "autoprefix": "postcss -c postcss.config.json",
```

Comment:
I've done prefixing like this since the CSS file we generate can be consumed as-is. Whatever we decide, we should be consistent across all the VUI repos.